test/e2e/test_sandbox.py

```
@@ -147,6 +148,46 @@ def test_sandbox_dryrun_opencode_custom_image():
+@skip_if_no_container
+def test_sandbox_dryrun_openclaw_custom_image():
+    """Custom --openclaw-image should appear in the OpenClaw container command."""
+    result = check_output(_dryrun_cmd("openclaw") + ["--openclaw-image", "myopenclaw:v2"])
+    assert "myopenclaw:v2" in result
+
+
+@pytest.mark.e2e
+@skip_if_no_container
+def test_sandbox_dryrun_openclaw_custom_port():
+    """Custom --openclaw-port should be used by the OpenClaw gateway and TUI."""
+    result = check_output(_dryrun_cmd("openclaw") + ["--openclaw-port", "19001"])
```

Ruff (0.15.9)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)